### PR TITLE
Update website 10/5/2018

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

Merge to update the website.
Preview at: https://vasilystrelyaev.github.io/testcafe/

Pay attention to redirects from 
* https://vasilystrelyaev.github.io/testcafe/documentation/
* https://vasilystrelyaev.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html

### CODE REVIEW

File: redirects.json [hunk 1]
```diff
@@ -1 +1 @@
-{"/documentation/":"http://devexpress.github.io/testcafe/documentation/getting-started/"}
\ No newline at end of file
+{"/documentation/test-api/selecting-page-elements/selectors.html":"http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/","/documentation/":"http://devexpress.github.io/testcafe/documentation/getting-started/"}
```
Comment (AlexKamaev) [inline redirects.json]: I think we need to set redirect for all deleted pages, not only for `selectors.html`

File: content/jquery-ui/AUTHORS.txt [hunk 2]
```diff
@@ -1,333 +1,333 @@
...
-Maksim Ryzhikov <rv.maksim@gmail.com>
-Amine HADDAD <haddad@allegorie.tv>
-Amanpreet Singh <apsdehal@gmail.com>
-Alexey Balchunas <bleshik@gmail.com>
-Peter Kehl <peter.kehl@gmail.com>
-Peter Dave Hello <hsu@peterdavehello.org>
-Johannes Schäfer <johnschaefer@gmx.de>
-Ville Skyttä <ville.skytta@iki.fi>
-Ryan Oriecuia <ryan.oriecuia@visioncritical.com>
+Authors ordered by first contribution
```
Comment (AlexKamaev) [inline content/jquery-ui/AUTHORS.txt]: Could we avoid comminting changes of these content files if they haven't changed?

File: example/index.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```
Comment (AlexKamaev) [inline example/index.html]: could we remove this unrequired changes?

**Comment by VasilyStrelyaev** [comment]
Changed line endings manually, but the next time we will build the website, these changes will appear again.

File: example/index.html (hunk 3 above)
Comment (VasilyStrelyaev) [reply example/index.html]: Not sure I understand what these changes mean.

**Pull request merged**